test/client.js

```diff
@@ -1272,11 +1272,4 @@ describe('Client', function() {
             });
         });
     });
-
-    after(function() {
-        var cache = new Client();
-
-        // Clean up all of the keys we created
-        return cache.deleteMulti(keys);
-    });
```

Comment:
Why was this removed? 

Without it this will create a mess, setting keys but never deleting them.